src/main/java/lotto/LottoTicket.java

```diff
@@ -17,9 +17,19 @@ public LottoTicket(int num) {
         return lottoList;
     }
 
-    public void compareWinList(List<Integer> winList) {
+    public void compareWinList(WinningLotto winningLotto, LottoNumber bonusNumber) {
+        List winList = winningLotto.value();
```

Comment:
```suggestion
        List<LottoNumber> winList = winningLotto.value();
```
제네릭에 타입은 꼭 명시해주셔야 합니다.